package.json

```diff
@@ -2,16 +2,20 @@
   "name": "poolifier",
   "version": "1.2.1",
   "description": "Library on top of node js worker threads that implement various worker pools type",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "lib/index.js",
   "scripts": {
-    "build": "npm install",
-    "test": "standard && nyc mocha --exit --timeout 20000 tests/*test.js ",
-    "debug-test": "mocha --inspect-brk --exit tests/*test.js ",
-    "demontest": "nodemon --exec \"npm test\"",
+    "build": "npm run build:clean && tsc",
+    "build:clean": "rimraf lib",
+    "test": "npm run build && nyc mocha --exit --timeout 20000 tests/*test.js ",
```

Comment:
The standard command is no more present into the test script, standard is a main component to ensure  the quality of this project

Reply:
If you expect standard to run without error in test script, then ts-standard should also be run here now.

Reply:
Yeah, I'm currently working on the branch
Give me some time to respond to all of the new stuff from this night ^^

Reply:
Ts-standard Is a good option if the corrente standard does not work proprrly for typescript :) It Is ok for me